A6-client-retries.md

```diff
@@ -134,7 +136,7 @@ Hedged requests are configured with the following parameters:
 }
 ```
 
-When a method has chosen a `hedgingPolicy`, the first RPC is sent immediately, as with a standard non-hedged call. After `hedgingDelayMs` has elapsed without a successful response, the second RPC will be issued. If neither RPC has received a response after `hedgingDelayMs` has elapsed again, a third RPC is sent, and so on, up to `maxRequests`. In the above configuration, after 1ms there would be one outstanding hedged RPC, after 501ms there would be two outstanding hedged RPCs, and after 1001ms there would be three outstanding hedged RPCs. As with retries, gRPC call deadlines apply to the entire chain of hedged requests. The time limit starts when the original RPC is issued, and if the deadline is reached all outstanding hedged requests are aborted.
+When a method has chosen a `hedgingPolicy`, the original RPC is sent immediately, as with a standard non-hedged call. After `hedgingDelayMs` has elapsed without a successful response, the first hedged RPC will be issued. If neither RPC has received a response after `hedgingDelayMs` has elapsed again, a second hedged RPC is sent, and so on, up to `maxHedgedRequests`. In the above configuration, after 1ms there would be one outstanding hedged RPC, after 501ms there would be two outstanding hedged RPCs, and after 1001ms there would be three outstanding hedged RPCs. As with retries, gRPC call deadlines apply to the entire chain of hedged requests. The time limit starts when the original RPC is issued, and if the deadline is reached all outstanding hedged requests are aborted.
```

Comment:
> The time limit starts when the original RPC is issued, and if the deadline is reached all outstanding hedged requests are aborted.

Maybe just a Go thing, but our timer starts when the context is created that specifies a deadline.  Even if a timeout is specified, Go converts that into a point-in-time deadline immediately (`WithTimeout(d)` shorthand for `WithDeadline(time.Now().Add(d))`).

If this is actually similar to how the other languages handle timeouts, then you may want to omit or more accurately specify when "the time limit starts".

Reply:
I think we should keep these in the terms used by the public APIs, so I think this wording is fine as-is

Reply:
As above - I was wrong here, gRPC allows setting deadlines in ways that do not start when the original RPC is issued, so Doug's right, we should remove this text.